### PR TITLE
Filter Handshake query params updated

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Ticket: [TM-1589](https://metaphase.atlassian.net/browse/TM-1589)

Dual Merge: [FE PR #852](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/852)

### CODE REVIEW

File: talentmap_api/fsbid/services/available_positions.py [hunk 1]
```diff
@@ -249,7 +249,7 @@ def convert_ap_query(query, cps_codes="OP,HS"):
         "request_params.page_index": int(query.get("page", 1)),
         "request_params.page_size": query.get("limit", 25),
         "request_params.freeText": query.get("q", None),
-        "request_params.cps_codes": services.convert_multi_value(cps_codes),
+        "request_params.cps_codes": services.convert_multi_value(query.get("cps_codes")),
```
Comment (mjoyce91) [inline talentmap_api/fsbid/services/available_positions.py:252]: Could we have some fallbacks and validation for this? I'm thinking:

- if query parameter is not provided, default to `OP,HS`
- only allow the values OP and HS

Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/available_positions.py:252]: Absolutely! Was wondering about that too. When testing it, I got all excited when I sent in the `HS` param and nothing came back. Then I sent in a random param and also got nothing back 💀

**Comment by mjoyce91** [comment]
@elizabeth-jimenez - something you'll need to make sure to do is add this to the list of allowed saved search parameters - otherwise trying to save a search with these new criteria will fail.

**Comment by elizabeth-jimenez** [comment]
> @elizabeth-jimenez - something you'll need to make sure to do is add this to the list of allowed saved search parameters - otherwise trying to save a search with these new criteria will fail.

@mjoyce91 ooo good call! Will do :)

**Comment by elizabeth-jimenez** [comment]
@mjoyce91 yaaayyy, now even if we send ', hs, ji, si.   ,     h o p  op, hs' as a value for `cps_codes` parameter, we should only get 'HS,OP'. I think 🤞 I did it properly 🤞

**Comment by mjoyce91** [comment]
@elizabeth-jimenez I'm gonna go ahead and merge this and worry about the saved searches separately

**Comment by elizabeth-jimenez** [comment]
> @elizabeth-jimenez I'm gonna go ahead and merge this and worry about the saved searches separately

Ok, sounds good! Good idea, since we found it was an older and bigger issue earlier today 👍

**Pull request merged**